frontend/simple/js/state.js

```diff
@@ -135,6 +188,7 @@ const actions = {
       }
       // TODO: verify each entry is signed by a group member
       await db.addLogEntry(contractId, entry)
+      entry.data.hash = hash
```

Comment:
There is no record of the hash of the event the other layers of the system to see. This was needed so that the invites could be responded to.